go/lang/security/audit/xss/no-interpolation-js-template-string.yaml

```diff
@@ -15,8 +15,8 @@ rules:
     - https://github.com/golang/go/issues/9200#issuecomment-66100328
     - https://blogtitle.github.io/robn-go-security-pearls-cross-site-scripting-xss/
   languages:
-  - none
+  - generic
   severity: WARNING
   patterns:
-  - pattern-regex: |-
-      `.*{{.*}}.*`
+  - pattern-inside: <script ...> ... ... ... ... ... </script>
```

Comment:
Why so many `...` here? Isn't one enough?

Reply:
spacegrep heuristically only spans 10 lines per ellipsis. So, this spans 50! 

![image](https://user-images.githubusercontent.com/2374948/99994951-94bd0480-2d6e-11eb-9664-40633d2a310b.png)

Reply:
I imagine script tags to be generally more than 10 lines, so I just guessed at 50. 🤷‍♂️